apps/env_service/app_config.json

```
@@ -6,7 +6,7 @@
     "adaptive_application": {
         "EnvApp": {
             "app": {
-                "bootMode": 0,
+                "bootMode": 4,
                 "parms": "-t -10- 100",
```

To było jako przykład nie musisz tam nic dawać
```suggestion
                "parms": "",
```

---

Możesz jak byś chciał przekazać parametry

---

wiem, ale jakos tak usuwac zapominam